src/main/resources/i18n/messages.properties

```diff
@@ -70,6 +70,7 @@ pet.view.edit.title=Edit a Pet
 pet.register=Register a pet
 pet.name=Pet's Name
 pet.birthDate=Date of Birth
+pet.birthTime=Time of Birth (optional)
 pet.dewormed=Dewormed
```

Comment:
Also this message value seems not used in the code.

Reply:
and done